src/error_retry_http_other.workflows.json

```diff
@@ -0,0 +1,62 @@
+{
+  "main": {
+    "steps": [
+      {
+        "read_item": {
+          "try": {
+            "steps": [
+              {
+                "callStep": null,
```

Comment:
Based on my read of https://cloud.google.com/workflows/docs/reference/syntax/steps, I think the Steps need to be contained inside the named block like so:

```
"steps": [
  {
    "callStep": {
      "call": "http.get",
      ...
}
```

this would be consistent with the other samples provided.

Reply:
Yes, you're right. (I did, however, use the `tojson` script to generate the JSON.) Do you want to push the change to my PR or should I close my PR and open another one?

Reply:
The nesting issue also appears in the YAML version (though it is harder to notice, IMO!).
you can just update this PR with additional commits and I'll be happy to re-review, no need to make a new PR.

Reply:
Ok, hope I did this correctly as I haven't done this before! Thanks and good catch btw :)